src/WorldWind.js

```diff
@@ -821,6 +847,8 @@ define([ // PLEASE KEEP ALL THIS IN ALPHABETICAL ORDER BY MODULE NAME (not direc
         WorldWind['ViewControlsLayer'] = ViewControlsLayer;
         WorldWind['WcsTileUrlBuilder'] = WcsTileUrlBuilder;
         WorldWind['WfsCapabilities'] = WfsCapabilities;
+        WorldWind['Wkt'] = Wkt;
+        WorldWind['WktType'] = WktType;
```

Comment:
We’ll need to export all WKT classes, otherwise apps can’t use them in extensions or to compose other functionality.